earthmover/nodes/destination.py

```diff
@@ -132,3 +136,79 @@ def render_row(self, row: pd.Series):
             raise
 
         return json_string
+
+
+class CsvDestination(Destination):
+    """
+
+    """
+    mode: str = 'csv'  # Documents which class was chosen.
+    allowed_configs: Tuple[str] = (
+        'debug', 'expect', 'show_progress', 'repartition', 'source',
+        'extension', 'header', 'separator', 'limit', 'keep_columns'
```

Comment:
I want to reiterate my opinion that `limit` and `keep_columns` should _not_ be part of destination configs. These are data transformations which should be done separately. We already have a `keep_columns` transformation operation, adding a `limit` operation would be simple (and, for performance reasons, should be done as far upstream as possible, not at the final destination).